docs/release_process.rst

```diff
@@ -105,73 +137,57 @@ notes file:
     Fixes
     -----
 
-Tag New Version
+Patching an Older Release
+"""""""""""""""""""""""""
```

Comment:
Is this header meant to be slightly smaller in text size? It looks a bit different than the "Stub Pending Notes File" and "Tag new version" sections that it falls in-between (probably difference between using ^^^ vs. """).

Reply:
I actually meant to remove the "Stub Pending Notes File" section, but apparently missed doing that, so I will remove that now (that's now controlled by the helper script). The "Patching an Older Release" is intended to be a sub-section under "Patch Release".

Reply:
In following the sphinx sections guidance ([here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections)), `"""` is listed as next in line after `^^^`.